src/main/java/org/wso2/carbon/connector/amazons3/utils/XmlUtil.java

```diff
@@ -27,4 +33,36 @@ public String convertToXml(Object source, Class... type) {
         }
         return result;
     }
+
+    public String convertToXml(Object source, String encoding, Class... type) {
+        String result = "";
+        StringWriter sw = new StringWriter();
+        try {
+            TransformerFactory transformerFactory = TransformerFactory.newInstance();
```

Comment:
If possible shall we add `FEATURE_SECURE_PROCESSING` as well? Refer https://github.com/wso2/micro-integrator/blob/master/components/org.wso2.micro.integrator.core/src/main/java/org/wso2/micro/core/util/CarbonUtils.java#L192

Reply:
added